H2MLauncher.UI/MapColorConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Globalization;
+using System.IO;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace H2MLauncher.UI
+{
+    public class MapColorConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is string folderName)
+            {
+                // Define the directory where map folders are located
+                string executableDirectory = AppDomain.CurrentDomain.BaseDirectory;

```

Comment:
As in the future we are introducing a settings menu to store the MWR location. 
Make this also check if it exists in the settings, otherwise try the local basedirectory.
So use this `_h2mLauncherSettings.MWRLocation` Check with string.IsNullOrEmpty

Like I have done here 

https://github.com/Bowhza/H2M-Launcher/blob/b2e74b7bcdb1e9e25c1cdd43c7c5de0789b18e09/H2MLauncher.Core/Services/H2MCommunicationService.cs#L105C43-L105C75